applications/nrf5340_audio/src/modules/lc3_file.c

```diff
@@ -12,6 +12,34 @@ LOG_MODULE_REGISTER(sd_card_lc3_file, CONFIG_MODULE_SD_CARD_LC3_FILE_LOG_LEVEL);
 
 #define LC3_FILE_ID 0xCC1C
 
+static void lc3_header_print(struct lc3_file_header *header)
+{
+	if (header == NULL) {
+		LOG_ERR("Nullptr received");
+		return;
+	}
+
+	LOG_DBG("File ID: 0x%04x", header->file_id);
+	LOG_DBG("Header size: %d", header->hdr_size);
+	LOG_DBG("Sample rate: %d Hz", header->sample_rate * 100);
+	LOG_DBG("Bit rate: %d bps", header->bit_rate * 100);
+	LOG_DBG("Channels: %d", header->channels);
+	LOG_DBG("Frame duration: %d us", header->frame_duration * 10);
```

Comment:
We could make a check on the scale again, since scale is different between samples :P

Reply:
Not sure I understand what you mean. Scale of frame duration? The *10 is taken from the header file that states that the frame duration has to be timed by 10 to get the correct duration in us. (it is actually ms * 100)